.github/workflows/velox_be.yml

```diff
@@ -82,11 +79,11 @@ jobs:
           cd /opt/gluten && \
           mvn clean install -Pspark-3.2 -Pspark-ut -Pbackends-velox -Prss -DargLine="-Dspark.test.home=/opt/spark322" -DtagsToExclude=org.apache.spark.tags.ExtendedSQLTest,io.glutenproject.tags.UDFTest,io.glutenproject.tags.SkipTestTags && \
           mvn test -Pspark-3.2 -Pbackends-velox -DtagsToExclude=None -DtagsToInclude=io.glutenproject.tags.UDFTest'
-      # Cpp micro benchmarks will use generated files from unit test in backends-velox module.
-      - name: Run micro benchmarks
-        run: |
-          docker exec ubuntu2004-test-$GITHUB_RUN_ID bash -c 'cd /opt/gluten/cpp/build/velox/benchmarks && \
-          ./generic_benchmark --with-shuffle --threads 1 --iterations 1'
+#       Cpp micro benchmarks will use generated files from unit test in backends-velox module.
+#      - name: Run micro benchmarks
+#        run: |
+#          docker exec ubuntu2004-test-$GITHUB_RUN_ID bash -c 'cd /opt/gluten/cpp/build/velox/benchmarks && \
+#          ./generic_benchmark --with-shuffle --threads 1 --iterations 1'
```

Comment:
benchmark is commented out, is this necessary?

Reply:
This benchmark generates ORC file with Arrow ORC file writer, but ORC is not compiled in Velox Arrow. We need to enable that first.

https://github.com/oap-project/gluten/pull/3341/files#diff-f1db35cb133919c45a122798963e00bebe0c957fcb8a552bd80951961fc0123fR28-R30